code/datums/mutations.dm

```diff
@@ -129,7 +129,8 @@
 	var/status = CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
 	owner.status_flags &= ~status
 	owner.update_body_parts()
-
+	owner_dna = owner.dna // fukken callbacks
```

Comment:
this... isn't required.
just put owner.dna where you did in the callback

(if your code compiled, it'd be functionally identical to just putting the var in the callback, no need for a temporary var.)